test/server_test.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+var fs = require('fs');
+var bodyparser = require('body-parser');
+var chai = require('chai');
+var chaihttp = require('chai-http');
+chai.use(chaihttp);
+var expect = chai.expect;
+
+require('../server.js');
+
+describe('Test that pages load', function () {
+  it('Should have status 200', function (done) {
+    chai.request('localhost:3000')
+    .get('/')
+    .end(function (err, res) {
+      expect(err).to.eql(null);
+      expect(res.status).to.eql(200);
+      done();
+    });
+  });
+});
+
+describe('Test Requests', function () {
+
```

Comment:
You need to have before/after statements that create files/unlink files so that your tests do not require you to create the files manually. If you need help, come find me.